### ISSUE TITLE
Max Feature Request - Start With 2 Weapons

### ISSUE THREAD

**Issue by DarthPickles666** [TheGameCreators/GameGuruRepo#5324]
So we currently have the ability to choose what weapon we start a level with and how many slots we want to use for weapons, but I think we should have the ability to choose 2 starting weapons. Like a Main and a Secondary weapon. Would this be easy enough to add? 

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/101579658/2b68c8d2-e886-49c5-93cd-cd644c8e555b)

**Comment by Necrym59** [comment]
A loadout screen is what your looking for, im working on one at the moment

**Comment by DarthPickles666** [comment]
A load out screen works, but I meant more like Halo or Call Of Duty where you're assigned 2 weapons for every level 😁

**Comment by Necrym59** [comment]
yes but how are they assigned? The loadout idea can be configured to have just the weapons designated available for use/choice would that suffice?

**Comment by DarthPickles666** [comment]
Ahhhh that makes a load more sense when you explain it that way!
